client-runtime/src/main/java/com/microsoft/rest/RestClient.java

```diff
@@ -41,6 +43,7 @@
     private final List<RequestPolicy.Factory> customPolicyFactories;
 
     private RestClient(RestClient.Builder builder) {
+        this.proxy = builder.proxy;
```

Comment:
Where do you actually use the proxy within the RxNettyAdapter?

Reply:
About to fix this. `httpClientBuilder.withProxy()` adds a Netty handler for dealing with the proxy. Would be nice to devise some test case that proves that the proxy works.